LocalStorage/LocalStorageManager/LocalStorageManagerProtocol.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LocalStorageManagerProtocol.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 06/04/2024.
+//
+
+import Foundation
+import Combine
+
+protocol LocalStorageManagerProtocol {
```

Comment:
```suggestion
protocol LocalStorageManager {
```
Personally, I prefer removing the protocol suffix for protocol, and instead add any suffix to the concrete type like 

Protocol --> Service
Concrete --> DefaultService: Service